src/prefect/_internal/compatibility/migration.py

```diff
@@ -30,12 +30,13 @@
 }
 
 REMOVED_IN_V3 = {
-    "prefect.deployments.deployments:Deployment": "Use 'flow.serve()', `flow.deploy()`, or `prefect deploy` instead.",
-    "prefect.deployments:Deployment": "Use 'flow.serve()', `flow.deploy()`, or `prefect deploy` instead.",
-    "prefect.filesystems:GCS": "Use 'prefect_gcp' instead.",
-    "prefect.filesystems:Azure": "Use 'prefect_azure' instead.",
-    "prefect.filesystems:S3": "Use 'prefect_aws' instead.",
-    "prefect.engine:_out_of_process_pause": "Use 'prefect.flow_runs.pause_flow_run' instead.",
+    "prefect.client.schemas.objects:MinimalDeploymentSchedule": "Use `prefect.client.schemas.actions.DeploymentScheduleCreate` instead.",
```

Comment:
These were inconsistent in using a single quote or a backtick for wrapping the command/new import. I've normalized them all to backticks here.